geode-dunit/src/main/java/org/apache/geode/management/internal/cli/commands/QueryCommandDUnitTestBase.java

```diff
@@ -310,4 +323,40 @@ public void setValue1(Value1 value1) {
       this.value1 = value1;
     }
   }
+
+  @Test
+  public void testSimpleQueryWithProxyRegion() {
+    server1.invoke(() -> setupReplicatedProxyRegion(DATA_REGION_WITH_PROXY_NAME));
+    server2.invoke(() -> setupReplicatedRegion(DATA_REGION_WITH_PROXY_NAME));
+    locator.waitUntilRegionIsReadyOnExactlyThisManyServers(DATA_REGION_WITH_PROXY_NAME_PATH, 2);
+
+    server1.invoke(() -> prepareDataForRegion(DATA_REGION_WITH_PROXY_NAME_PATH));
+
+    String member = getHostingMember();
+    Random random = new Random(System.nanoTime());
+    int randomInteger = random.nextInt(COUNT);
+    String query = "query --member=" + member
+        + " --query=\"select ID , status , createTime , pk, floatMinValue from "
+        + DATA_REGION_WITH_PROXY_NAME_PATH + " where ID <= " + randomInteger
+        + "\" --interactive=false";
+
+    CommandResult commandResult = gfsh.executeCommand(query);
```

Comment:
`executeCommand()` is deprecated. You can use `executeAndAssertThat(query).getCommandResult()` instead.

Reply:
+1